awx/main/redact.py

```diff
@@ -1,12 +1,13 @@
 import re
 import urlparse
+from six import u
 
 REPLACE_STR = '$encrypted$'
 
 
 class UriCleaner(object):
     REPLACE_STR = REPLACE_STR
-    SENSITIVE_URI_PATTERN = re.compile(ur'(\w+:(\/?\/?)[^\s]+)', re.MULTILINE)  # NOQA
+    SENSITIVE_URI_PATTERN = re.compile(u(r'(\w+:(\/?\/?)[^\s]+)'), re.MULTILINE)  # NOQA
```

Comment:
Would the following code work?
```python
    SENSITIVE_URI_PATTERN = re.compile(r'(\w+:(\/?\/?)[^\s]+)', re.UNICODE|re.MULTILINE)
```

Reply:
Hi, I was thinking the same thing. Does the regex need to be declared unicode, would have to do more research on that or develop some test cases.